partner_coc/i18n/partner_coc.pot

```diff
@@ -19,10 +19,8 @@ msgid "Chamber Of Commerce Registration Number"
 msgstr ""
 
 #. module: partner_coc
-#: model:ir.model.fields,field_description:partner_coc.field_res_partner_alias_coc_registration_number
-#: model:ir.model.fields,field_description:partner_coc.field_res_partner_coc_registration_number
-#: model:ir.model.fields,field_description:partner_coc.field_res_users_coc_registration_number
-msgid "CoC Registration Number"
+#: field:res.partner,coc_registration_number:0
+msgid "Chamber of Commerce"
```

Comment:
I think if you change the wording here, you should be nice to the other translations and change the English term there too

Reply:
I reversed the unneeded change in this string.